frontend/public/scss/product-faculty-members.scss

```diff
@@ -19,16 +19,17 @@
 
     width: calc(50% - 44px);
     margin: 0 22px 24px;
+    min-width: min-content;
 
     .member-card {
       padding: 24px;
-      overflow: hidden;
       border-radius: 4px !important;
       box-shadow: 0 1px 0 0 rgb(0 0 0 / 12%);
+      display: flex;
```

Comment:
This has side effect on Mobile view for both new design and production course page. Can you confirm?

![Pasted Graphic 3](https://github.com/mitodl/mitxonline/assets/3138890/03b2e5eb-2be1-4f63-9ba8-5d85915710d1)
vs production
![Pasted Graphic 4](https://github.com/mitodl/mitxonline/assets/3138890/c9f6bc90-90fa-4657-82cc-fb32c54c43e7)

Reply:
good catch!